app/views/editions/workflow/review.html.erb

```diff
@@ -7,8 +7,10 @@
   } %>
 <% end %>
 
-<% content_for :title_side do %>
-  <%= render "shared/preview_link", edition: @edition %>
+<% if @edition.schema.embeddable_as_block? %>
```

Comment:
Are "composed block" and "embeddable as block" synonyms? If so, should we refactor the naming to have `schema.composed_block?`?